src/constants.ts

```diff
@@ -18,6 +18,7 @@ export const ANIME_REGEX =
 	/^(?:\[(?<group>.*?)\][_.\s-]?)?(?:\[?(?<title>.+?)[_.\s-]?(?:\(?(?:\d{1,2}(?:st|nd|rd|th))?\s?Season)?[_.\s-]?\]?)(?:[([~/|-]\s?(?!\d{4})(?<altTitle>.+?)[)\]~-]?\s?)?[_.\s-]?(?:\(?(?<year>(?:19|20)\d{2})\)?)?[[_.\s-](?:S\d{1,2})?[_.\s-]{0,3}(?:#|EP?|(?:SP))?[_.\s-]{0,3}(?<release>\d{1,4})(?:v\d)?[\])_.\s-]/i;
 export const RELEASE_GROUP_REGEX =
 	/(?<=-)(?:\W|\b)(?!(?:\d{3,4}[ip]))(?!\d+\b)(?:\W|\b)(?<group>[\w ]+?)(?:\[.+\])?(?:\))?(?=(?:\.\w{1,5})?$)/i;
+export const RP_REGEX = /\b(?<type>(REPACK|PROPER)\d?)\b/i;
```

Comment:
```suggestion
export const RP_REGEX = /(?:\b|\d)(?<type>(REPACK\d?|PROPER\d?|v\d))\b/i;
```
I think we should also support anime style of vX. This is sometimes vX.Y but vX should be enough. Preceding character can be digit like `S01E05v2`.

Reply:
Did you test this? it looks fine to me but we'd want to cover conservatively rather than too much which could lead to false-negatives.

Reply:
Let me know if this works...
```ts
export const RP_REGEX =
	/(\b(?<type>(REPACK|PROPER)\d?))|(?<arrtype>(Proper|v\d)\d?)\b/;
```
Given the context of the logic changes I've made since you reviewed....

Reply:
Like this?

```ts
export const RP_REGEX =
	/(\b(?<type>(REPACK|PROPER|\d\v\d)\d?))|(?<arrtype>(Proper|\sv\d)\d?)\b/;```

check within the context of the latest push

Reply:
Moved to: https://github.com/cross-seed/cross-seed/pull/666#discussion_r1581380270